### PR TITLE
Frontend base

### PR DESCRIPTION
**Pull request by viivi-uhari**
A simple React app for frontend development. 
Related issues: #8 #9

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+$ npm install
+```
+Start the program in browser: 
+```
+$ npm start
+```
+Runs the app in development mode. Open [http://localhost:3005](http://localhost:3005) to view it in your browser.
+
+## Testing
+`npm test` \
```
Comment (no-comma) [inline client/README.md:18]: Maybe add `$` (or `#` if root) in front of commands for the sake of convention.

File: client/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Install node modules: 
+```
+$ npm install
+```
+Start the program in browser: 
+```
+$ npm start
+```
+Runs the app in development mode. Open [http://localhost:3005](http://localhost:3005) to view it in your browser.
```
Comment (no-comma) [inline client/README.md:15]: Just having http://localhost:3005 on its own should already make it a link.

File: client/src/tests/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from '../App';
+
+test('renders learn react link', () => {
+  render(<App />);
```
Comment (no-comma) [inline client/src/tests/App.test.js:5]: Running `$ npm test` produces warnings and errors for me and this test fails. Is this expected?

**Review by no-comma — COMMENTED** [review]
There's some inconsistent indentation here, but we haven't decided on a style guide yet to begin with so it's not a big deal now.

**Review by kiiamakir — APPROVED** [review]
README syntax errors have been fixed and npm test now works and no longer produces warnings.

**Review by marillam — APPROVED** [review]
Looks good. Src structure is clean and port has been switched to 3005 as discussed. All needed packages are included.

**Pull request merged**